core/governance/market.go

```diff
@@ -615,6 +620,24 @@ func validateNewMarketChange(
 	return types.ProposalErrorUnspecified, nil
 }
 
+func validateUseOfEthOracles(terms *types.NewMarket, netp NetParams) (types.ProposalError, error) {
```

Comment:
This validation could probably be done here in the processor: https://github.com/vegaprotocol/vega/blob/develop/core/processor/abci.go#L1107

This is where we were doing the CanPropose ones, the advantage is that as the check is done in the tendermint checkTx, the transaction is rejected as soon as it get submitted, and never even get onto a block.